scrapers/hussiemodels.yml

```diff
@@ -0,0 +1,56 @@
+name: hussiemodels
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - hussiepass.com/trailers/
+      - hookuphotshot.com/trailers
+      - seehimfuck.com/trailers
+      - povpornstars.com
+      # Removes all emoticons from Title, Description, Performers and Tags.
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //div[@class="videoDetails clear"]/h3
+     Date:
+        selector: //div[@class='videoInfo clear']/p[contains(span,"Date Added:")]/text()
+        postProcess:
+          - replace:
+              - regex:
+                with:
```

Comment:
leftover from copy paste?

Reply:
Did not even notice that yeah would have been a left over.